src/array/libra_partition.cc

```diff
@@ -258,7 +258,7 @@ void LibraVertexCut(
       }
     }
   }
-  delete cache;
+  delete[] cache;
```

Comment:
Ooof, I like that compiler warnings caught this.

Reply:
Hooray for clang! 😁